src/main/java/seedu/smartnus/logic/parser/ParserUtil.java

```diff
@@ -238,8 +240,8 @@ public static Choice parseAnswerForEdit(String answer) throws ParseException {
     public static Set<Choice> parseTrueFalseAnswer(String answer) throws ParseException {
         requireNonNull(answer);
         String trimmedAnswer = answer.trim();
```

Comment:
Why not just add `.toUpperCase()` instead?

Reply:
Changed